board-layout.json

```diff
@@ -195,13 +195,36 @@
         "⛰️R": "C",
         "⛰️S": "H",
         "⛰️T": "A",
-        "⛰️U": "D"
-    },
-    "missing": [
```

Comment:
please leave an empty list here or remove the code in islet.py that fallbacks when data is missing

Reply:
Fixed!